lib/status-bar.js

```diff
@@ -506,6 +506,7 @@ class StatusBar extends View {
 
     const fromIndex = parseInt(dataTransfer.getData('from-index'))
     const view = this.terminalViews[fromIndex]
+    if (!view) return
```

Comment:
even with the terminals in the bottom panel, when moved, I see an error notification because `view` is undefined here

Reply:
I dont see an error (this comment is without this pr in), but say opening a file in Atom and moving the panel next to the tab, the open file tab, the tab disappears. Terminal is then in a tab but I see no errors. Ill try later with macOS virtualbox also.

Reply:
Moving dock theres an error.

[Enter steps to reproduce:]

![generic](https://user-images.githubusercontent.com/31389848/69341441-def63c00-0c69-11ea-8db2-b426ea87b8e4.gif)

**Atom**: 1.41.0 x64
**Electron**: 4.2.7
**OS**: Ubuntu 18.04.3
**Thrown From**: [terminus](https://github.com/bus-stop/terminus) package 4.0.2

### Stack Trace

Uncaught TypeError: Cannot read property 'destroy' of undefined

```
At /home/terminus/Documents/Example/terminus/lib/view.js:750

TypeError: Cannot read property 'destroy' of undefined
    at TerminusView.toggleTabView (/home/terminus/Documents/Example/terminus/lib/view.js:750:18)
    at StatusBar.onDropTabBar (/home/terminus/Documents/Example/terminus/lib/status-bar.js:514:10)
    at HTMLUListElement.tabBar.on.event (/home/terminus/Documents/Example/terminus/lib/status-bar.js:183:39)
    at HTMLUListElement.dispatch (/home/terminus/Downloads/terminus/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLUListElement.elemData.handle (/home/terminus/Downloads/terminus/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
```

### Non-Core Packages

```
terminus 4.0.2 
```

FYI this works without being on dock

![generic](https://user-images.githubusercontent.com/31389848/69341707-5a57ed80-0c6a-11ea-8d7a-e1a4fa7fce5a.gif)

Reply:
It looks like the toolbar on windows hides once the terminal is too big.
![terminus1](https://user-images.githubusercontent.com/97994/69371554-aeac9e80-0c65-11ea-80a2-39e3a237fb7c.gif)

Reply:
Yea the input button should be in the status bar while using the dock. I hadn't gotten to that yet.

I think we should move forward with the strategy in #15

Reply:
That sounds good. =)